src/sass/_variables.scss

```diff
@@ -100,6 +100,7 @@ $content-mobile-wrap-pr : 1rem;
 $content-mobile-wrap-pt : 2rem;
 $content-mobile-wrap-pb : $layout-padding;
 $content-mobile-wrap: $content-mobile-wrap-pt $content-mobile-wrap-pr $content-mobile-wrap-pb $content-mobile-wrap-pl;
+$content-mobile-wrap-sb-top: calc(#{$top-nav-height} + #{$layout-padding+20});
```

Comment:
mobile view - padding right is ugly

Emulate iPhone 6 in chrome both for this PR and vuestic.epicmax.co